docs/src/conventions.md

```diff
@@ -0,0 +1,123 @@
+These checks are required for merging any PR.
+
+For Rust, we use [rustfmt](https://github.com/rust-lang/rustfmt)
+which is configured via the [rustfmt.toml](https://github.com/model-checking/kani/blob/main/rustfmt.toml) file.
+We are also in the process of enabling `clippy`.
+Because of that, we still have a couple of lints disabled (see [.cargo/config](https://github.com/model-checking/kani/blob/main/.cargo/config.toml) for the updated list).
+
+We also have a bit of C and Python code in our repository.
+For C we use `clang-format` and for Python scripts we use `autopep8`.
+See [.clang-format](https://githubcom/model-checking/kani/blob/main/.clang-format)
+and [pyproject.toml](https://github.com/model-checking/kani/blob/main/scripts/pyproject.toml)
+for their configuration.
+
+
+### Exceptions
+
+We recognize that in some cases, the formatting and lints automation may not be applicable to a specific code.
+In those cases, we usually prefer explicitly allowing exceptions by locally disabling the check.
+E.g., use `#[allow]` annotation instead of disabling a link on a crate or project level.
+
+### Copyright notice
+
+All source code files begin with a copyright and license notice. If this is a new file, please add the following notice:
+
+```rust
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+```
+
+When modifying a file from another project, please keep their headers as is and append the following notice after them:
+
+```rust
+// ... existing licensing headers ...
+
+// Modifications Copyright Kani Contributors
+// See GitHub history for details.
+```
+
+Note: The comment escape characters will depend on the type of file you are working with. E.g.: For rust start the 
+header with `//`, but for python start with `#`.
+
+We also have automated checks for the copyright notice.
+There are a few file types where this rule doesn't apply.
+You can see that list in the [copyright-exclude](
+https://github.com/model-checking/kani/blob/main/scripts/ci/copyright-exclude) file.
+
+
+## Code for soundness
+
+We are developing Kani to provide assurance that critical Rust components are verifiably free of certain classes of
+security and correctness issues.
+Thus, it is critical that we provide a verification tool that is sound.
+For the class of errors that Kani can verify, we should not produce a “No Error” result if there was in fact an
+error in the code being verified, i.e., it has no
+“False Negatives”.
+
+Because of that, we bias on the side of correctness.
+Any incorrect modeling
+that may trigger an unsound analysis that cannot be fixed in the short term should be mitigated.
+Here are a few ways how we do that.
+
+### Compilation errors
+
+Make sure to add user-friendly errors for constructs that we can't handle.
+For example, Kani cannot handle panic unwind strategy, and it will fail compilation if the crate uses this
```

Comment:
```suggestion
For example, Kani cannot handle the panic unwind strategy, and it will fail compilation if the crate uses this
```
Optionally, you can link to its subsection in limitations.